scripts/remotes.lua

```diff
@@ -369,7 +591,7 @@ function remotes.cluster_targeting(player, surface, requested_position, remote_p
 
   -- Create list of target positions.
   for _, entity in pairs(target_entities) do
-    table.insert(targets, entity.position)
+    table.insert(targets, entity.bounding_box)
```

Comment:
Hm... So, on the second pass, I am actually trying to understand one thing - is the `circle_centroid`  implementation needed at all? Unless I am missing something, the `center_point` function will ever only accept bounding boxes (rectangles)?

Reply:
The `assign_targets` function is run in 'reverse' to assign clusters to clusters to order to merge clusters together. So in that cases it's circles to circles. See the noted steps 3 & 4.